cli/dcoscli/service/main.py

```diff
@@ -123,3 +146,170 @@ def _shutdown(service_id):
 
     mesos.DCOSClient().shutdown_framework(service_id)
     return 0
+
+
+def _log(follow, lines, config_file, service, file_):
+    """Prints the contents of the logs for a given service.  The service
+    task is located by first identifying the marathon app running a
+    framework named `service`.
+
+    :param follow: same as unix tail's -f
+    :type follow: bool
+    :param lines: number of lines to print
+    :type lines: int
+    :param config_file: SSH config file.  Used for marathon.
+    :type config_file: str | None
+    :param service: service name
+    :type service: str
+    :param file_: file path to read
+    :type file_: str
+    :returns: process return code
+    :rtype: int
+    """
+
+    lines = util.parse_int(lines)
+
+    if service == 'marathon':
+        if file_:
+            raise DCOSException('The <file> argument is invalid for marathon.'
+                                ' The systemd journal is always used for the'
+                                ' marathon log.')
```

Comment:
Okay. I am going to work on a proposal for more detail documentation of command when the user types: `dcos <commmand> --help`